### ISSUE TITLE
Allow user to change default EPUB font

### ISSUE THREAD

**Issue by ToaKraka** [sumatrapdfreader/sumatrapdf#3138]
When an EPUB document's CSS stylesheet does not specify a `font-family`, SumatraPDF uses a serif font as the default. However, some people prefer to read a sans-serif font, and others may prefer to use some specific font that they have installed on their computer, without being forced to manually add that font to the EPUB file's stylesheet.

Alternatively, if adding this setting to the settings dialog would be too complicated, then maybe the entire default "user-agent stylesheet" could be exposed in `%LOCALAPPDATA%\SumatraPDF`, for the user to edit.

**Comment by RoWaBe** [comment]
Yes yes yes. Non-Serif fonts look much more clear especially at smaller/LoRes screens.
This is the most-wanted feature for me for the moment.
